src/main/proto/google/rpc/context/attribute_context.proto

```
@@ -0,0 +1,345 @@
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.rpc.context;
```

Buf (1.54.0)

Files with package "google.rpc.context" must be within a directory "google/rpc/context" relative to root but were in directory "src/main/proto/google/rpc/context".

(PACKAGE_DIRECTORY_MATCH)